### PR TITLE
Fix bug 1671295 - Fix the User Admin form and show the email field on…

### PR DESCRIPTION
**Pull request by jotes**

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -1525,7 +1524,9 @@ def available_locales_list(self):
 
 class UserProfile(models.Model):
     # This field is required.
-    user = models.OneToOneField(User, models.CASCADE, related_name="profile")
+    user = models.OneToOneField(
+        User, models.CASCADE, related_name="profile", primary_key=True
```
Comment (jotes) [inline pontoon/base/models.py:1528]: `primary_key=True` is required to make Django to figure out the relation between the objects and remove the IntegrittyError

File: pontoon/base/admin.py [hunk 2]
```diff
@@ -43,6 +43,13 @@ class UserAdmin(AuthUserAdmin):
     list_filter = ("is_staff", "is_superuser")
     inlines = (UserProfileInline,)
 
+    add_fieldsets = (
+        (None, {
+            'classes': 'wide',
+            'fields': ('username', 'email', 'password1', 'password2')
```
Comment (jotes) [inline pontoon/base/admin.py:49]: I added this for the sake of UI because email is the first field on the  users list view.

**Comment by jotes** [comment]
@mathjazz could you r?

**Review by mathjazz — APPROVED** [review]
Works great! Thanks!

**Pull request merged**